api/src/main/resources/application.yml

```diff
@@ -16,4 +16,4 @@ spring:
 
 tanagra:
   underlay:
-    underlay-yaml-files: ["underlays/aou_synthetic.yaml", "underlays/synpuf.yaml"]
+    underlay-yaml-files: ["underlays/aou_synthetic.yaml", "underlays/synpuf.yaml", "underlays/sd.yaml"]
```

Comment:
I don't expect our dev instance to be able to load this (I'd say it **shouldn't** be able to), should we be splitting our the configs for local vs dev ?

Reply:
Yes, we will want to split them. I've added a TODO here.